Test/data/es/statusLogMapping.json

```diff
@@ -0,0 +1,18 @@
+{
+  "_all": {
+    "enabled": false
+  },
+  "_id": {
+    "store": "yes",
+    "index": "not_analyzed"
+  },
+  "properties": {
+    "date": {
```

Comment:
I used the same format as the existing mapping.json file in the bundle, which also excluded `id` from the properties.